rainbows.rb

```diff
@@ -26,4 +28,71 @@ def mix_colors(color1,color2)
 end
 
 new_color = mix_colors(red,green)
-puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+# puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+
+rainbow = []
+rainbow[0] = {
+  red: {
+    r: 255,
+    g: 0,
+    b: 0
+  }
+}
+
+rainbow[1] = {
+  orange: {
+    r: 255,
+    g: 127,
+    b: 0
+  }
+}
+
+rainbow[2] = {
+  yellow: {
+    r: 255,
+    g: 255,
+    b: 0
+  }
+}
+
+rainbow[3] = {
+  green: {
+    r: 0,
+    g: 255,
+    b: 0
+  }
+}
+
+rainbow[4] = {
+  blue: {
+    r: 0,
+    g: 0,
+    b: 255
+  }
+}
+
+rainbow[5] = {
+  indigo: {
+    r: 75,
+    g: 0,
+    b: 130
+  }
+}
+
+rainbow[6] = {
+  violet: {
+    r: 143,
+    g: 0,
+    b: 255
+  }
+}
+
+rainbow.each do |hash|
+  r = hash.first[1][:r]
+  g = hash.first[1][:g]
+  b = hash.first[1][:b]
+  puts Rainbow(hash.first[0].to_s).color(r, g, b)
```

Comment:
Yes!! This is what I couldn't figure out on my own! But that's what I wanted you all to do!